packages/angular_devkit/core/src/workspace/workspace-schema.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+// tslint:disable
+export interface WorkspaceSchema {
+  /**
+   * Link to schema.
+   */
+  $schema?: string;
+  /**
+   * Workspace Schema version.
+   */
+  version: number;
+  /**
+   * New project root.
+   */
+  newProjectRoot?: string;
+  /**
+   * Tool options.
+   */
+  cli?: {
+    /**
+     * Link to schema.
+     */
+    $schema?: string;
```

Comment:
I thought these did something but they don't. Only the toplevel `$schema` works. Sorry.

Reply:
This interface is only a helper, so no worries.